app.midterm.MidtermDesktop.yml

```diff
@@ -0,0 +1,68 @@
+app-id: app.midterm.MidtermDesktop
+runtime: org.freedesktop.Sdk
+runtime-version: "20.08"
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+command: midterm
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=host
```

Comment:
What does it need host access for?

Reply:
I tried to remove `--filesystem=host` but then I cannot pick images from file chooser so I think it's required.

Reply:
Is there a more constrained set of folders than "anywhere on the computer" that would make sense? e.g. just the home directory or just the xdg-* folders for instance.

Reply:
Thanks @nedrichards, I use `--filesystem=home` now. I think it's good enough.